managedApplication/parameters-template.json

```diff
@@ -4,5 +4,6 @@
     "dbServerType":         { "value": "mysql" },
     "fileServerType":       { "value": "nfs" },
     "autoscaleVmSku":       { "value": "Standard_DS1_v2" },
-    "fileServerDiskCount":  { "value": 2 }
+    "fileServerDiskCount":  { "value": 2 },
+    "azureSearchSwitch":    { "value": true }
```

Comment:
Probably the same. Please turn this off to false. Better yet, we can depend on the default value, so just remove this.